neuro_san/service/http/server/http_server.py

```diff
@@ -126,12 +119,21 @@ def make_app(self, requests_limit: int, logger: EventLoopLogger):
         Construct tornado HTTP "application" to run.
         """
         request_data: Dict[str, Any] = self.build_request_data()
-        health_request_data: Dict[str, Any] = {
-            "forwarded_request_metadata": self.forwarded_request_metadata
+        live_request_data: Dict[str, Any] = {
+            "forwarded_request_metadata": self.forwarded_request_metadata,
+            "server_status": self.server_status,
+            "op": "live"
+        }
+        ready_request_data: Dict[str, Any] = {
+            "forwarded_request_metadata": self.forwarded_request_metadata,
+            "server_status": self.server_status,
+            "op": "ready"
```

Comment:
Maybe call these all "initialize_data" so it's clearer what's happening.
This is probably the 8th time I've scratched my head saying "WTF is happening here?"
... and then I remember the initialize() call.